help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage and Activate Dataset Destinations]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+#### New account {#new-account}
+
+| Field | Description |
+| --- | --- |
+| [!UICONTROL Application ID] | The [!DNL Algolia] application ID is a unique identifier assigned to your [!DNL Algolia] account. |
+| [!UICONTROL API Key] | The [!DNL Algolia] API Key is a credential used to authenticate and authorize API requests to [!DNL Algolia]'s search and indexing services. |
+
+For more information on these credentials, see the [!DNL Algolia] [authentication documentation](https://www.algolia.com/doc/tools/cli/get-started/authentication/).
+
+![New Account](../../assets/catalog/personalization/algolia-user-segments/connection.png)
+
+After connection to destination is successful, enter the below information to complete the account creation.
+
+| Field | Description |
+| --- | --- |
+| [!UICONTROL Name] | Fill in the preferred name for this destination. |
+| [!UICONTROL Description] | Short explanation of the destination's purpose. |
+| [!UICONTROL Region] | The options are **US** or **EU**. Select the region where the customer data is stored. |
+
+![Account details](../../assets/catalog/personalization/algolia-user-segments/account.png)
+
+### Existing account {#existing-account}
+
+Accounts already defined using the [!DNL Algolia User Segments] destination appear in a list pop-up. When selected, you can see details on the account in the right rail. View the example from the UI, when you navigate to **[!UICONTROL Destinations]** > **[!UICONTROL Accounts]**:
+
+![Existing Account](../../assets/catalog/personalization/algolia-user-segments/existing-account.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Data usage and governance {#data-usage-governance}
```

Comment:
1. There is no need for the screenshot in this section. This section applies to all destinations and doesn't require any updates. Use it as you see it in the template https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#data-usage-governance
2. Not sure why you moved this section here. Please leave this section at the end of the page as you see in the template.

Do not move around headings from the template structure.